src/govuk/components/notification-banner/notification-banner.test.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-env jest */
+
+const configPaths = require('../../../../config/paths.json')
+const PORT = configPaths.ports.test
+
+const baseUrl = 'http://localhost:' + PORT
+
+describe('/components/notification-banner/with-type-as-success', () => {
+  describe('when JavaScript is available', () => {
```

Comment:
We could probably remove this `describe` – I don't think we generally include this unless we also have tests for the non-JS fallback behaviour.

Reply:
Good point, removed.